R/constants.R

```diff
@@ -82,7 +82,7 @@ standard_fields <- c(
 
 cran_valid_fields <- c(
   standard_fields,
-  "^(X-CRAN|Repository/R-Forge)",
+  "^(?:X-CRAN|Repository/R-Forge)",
```

Comment:
YEs, this probably makes sense, in case anybody uses these in regular expressions.